encoding/gjson/gjson_api_new_load.go

```diff
@@ -228,7 +228,7 @@ func IsValidDataType(dataType string) bool {
 func checkDataType(content []byte) string {
 	if json.Valid(content) {
 		return "json"
-	} else if gregex.IsMatch(`^<.+>[\S\s]+<.+>$`, content) {
+	} else if gregex.IsMatch(`^<.+>[\S\s]+<.+>[ \n\r\t]*$`, content) {
```

Comment:
@coolhihi Maybe it could use `^<.+>[\S\s]+<.+>[\s]*$` instead better here?

Reply:
you're right. `[\s]*`

Reply:
`\s*`